webapp-django/crashstats/supersearch/static/supersearch/js/lib/dynamic_form.js

```diff
@@ -124,6 +123,7 @@
                 if (onReadyCallback) {
                     onReadyCallback();
                 }
+                $('.loader', container).remove();
```

Comment:
I don't like this. It just doesn't feel right. This function calls "onReadyCallback()" but it's clearly not ready if it does more. Just the name implies that we're lying :)

It also doesn't feel right to do more in a function after you've "exited" by calling the callback. 

What could happens is this:

```javascript
showLoader()
fetch(...).then(r => {
    someCallback(r)
    hideLoader()
})
function someCallback(data) {
    showTable(data)
}
```
Your DOM will then be...
```
<!-- frame 1 -->
<div class="loading">Please wait</div>
```
```
<!-- frame 2 -->
<div class="loading">Please wait</div>
<table>
  data
</table>
```
```
<!-- frame 3 -->
<table>
  data
</table>
```